Blimp/radio.cpp

```diff
@@ -74,8 +74,8 @@ void Blimp::read_radio()
     }
 
     const uint32_t elapsed = tnow_ms - last_radio_update_ms;
-    // turn on throttle failsafe if no update from the RC Radio for 500ms or 2000ms if we are using RC_OVERRIDE
-    const uint32_t timeout = RC_Channels::has_active_overrides() ? FS_RADIO_RC_OVERRIDE_TIMEOUT_MS : FS_RADIO_TIMEOUT_MS;
+    // turn on throttle failsafe if no update from the RC Radio for RC_FS_TIMEOUT seconds or 1000ms if we are using RC_OVERRIDE
+    const uint32_t timeout = RC_Channels::has_active_overrides() ? FS_RADIO_RC_OVERRIDE_TIMEOUT_MS : rc().get_fs_timeout_ms();
```

Comment:
This is fine but I think we should make it the same as Copter which is simply, "if (elapsed_ms < rc().get_fs_timeout_ms()) {"